puppeteer.setup.js

```diff
@@ -7,6 +10,8 @@ import { setDefaultOptions } from 'expect-puppeteer';
 jest.setTimeout(30 * 1000);
 setDefaultOptions({ timeout: 3 * 1000 });
 
+jest.retryTimes(4);
```

Comment:
Add retry 4 times for a total test run of 5 times before jest reporting a failed test. This is useful for avoiding minor flakiness when doing UI testing

Reply:
I was finding it relatively hard to force an error (_I was trying to do so by mucking with CSS, to cause a noticeable visual change)_, I think partially because retries were set high enough that it was taking quite some time to exit.

In deciding on your retry/timeout settings, mind doing the same, just to confirm that an expected failure occurs in a reasonable period of time, and bubbles appropriately.